doc/rst/technotes/globalvars.rst

```diff
@@ -0,0 +1,47 @@
+.. _readme-globalvars:
+
+==================================
+Global Variables
+==================================
+
+.. warning:: The variables defined here are not part of Chapel Standard and 
+             they can be changed, renamed, or removed at any point in the future.
+
+The Chapel Compiler defines some variables that are available to all Chapel programs. 
+These variables are used by the compiler itself and the standard Chapel libraries/packages 
+to provide some features/optimizations based on compilation flags. 
+All of these variables are compile-time constants.
+
+Variables
+---------
+
+**boundsChecking**
+
+    This variable can be used to check if ``bounds-checks`` are enabled or not. 
```

Comment:
I'd either refer to the compiler commandline option with dashes `--bounds-checks` as singular, or refer to "bounds checks" as plural.  This variable is about the commandline option, not general bounds checking, so I'd say it should be "... can be used to check if ``--bounds-checks`` is enabled or not."  or "if the bounds checking controlled by ``--bounds-checks`` are enabled or not.

Or even strike the "to check if", and say "This variable is true if ...", and drop the "or not" at the end.

Similarly for the other `--*-checks` command-line options.